rh/views.py

```diff
@@ -1,18 +1,74 @@
-from email import message
-from django.shortcuts import render
+from functools import cache
+import pandas as pd
+import datetime
+import sqlite3
+from multiprocessing import context
+from tkinter import E
```

Comment:
I imagine you forgot to remove this one ?

Reply:
Yes I forgot removing these xD